src/server/server.ts

```diff
@@ -1,127 +1,24 @@
-import './server-assets'
-import type { IServerAPI } from 'vs/server/node/remoteExtensionHostAgentServer'
-import { createServer } from 'vs/server/node/server.main.js'
-import { buildHelpMessage, buildVersionMessage, parseArgs } from 'vs/platform/environment/node/argv'
-import { ServerParsedArgs, serverOptions } from 'vs/server/node/serverEnvironmentService'
-import product from 'vs/platform/product/common/product'
-import { AddressInfo, ListenOptions, Socket } from 'net'
-import http from 'http'
+import { IProductService } from '../services'
```

Comment:
please import from the module defining it directly

services.ts should not be included by the server